src/wmnds/patterns/header-v2/_header-v2.scss

```diff
@@ -0,0 +1,678 @@
+//transition variables
```

Comment:
Can this file be split up a bit more so it is easier to digest?

Also the mobile navigation kicks in too early...It shouldn't appear until on mobile (less than $breapoint-md).